mava/utils/make_env.py

```diff
@@ -26,12 +26,13 @@
 )
 from jumanji.wrappers import AutoResetWrapper
 
+import mava.utils.jumanji_registry as jumanji_registry
 from mava.wrappers.jaxmarl import JaxMarlWrapper
 from mava.wrappers.jumanji import LbfWrapper, RwareWrapper
 from mava.wrappers.shared import AgentIDWrapper, GlobalStateWrapper, LogWrapper
 
 # Registry mapping environment names to their generator and wrapper classes.
-_jumanji_registry = {
+_jumanji_env_class = {
```

Comment:
I can't really think of a nice name for this :thinking: 

```suggestion
_jumanji_wrappers_and_args = {
```

Reply:
Same here and I found no short name 🤔 but this seems more descriptive thank you 🙌